pkg/pgmodel/migrations/sql/idempotent/base.sql

```diff
@@ -1193,6 +1233,41 @@ CREATE OR REPLACE VIEW SCHEMA_INFO.label AS
       AS values
   FROM SCHEMA_CATALOG.label_key lk;
 
+CREATE OR REPLACE VIEW prom_info.system_stats AS
+    SELECT
+    (
+        SELECT SUM(child.reltuples) FROM pg_inherits
+            JOIN pg_class parent ON pg_inherits.inhparent = parent.oid
+            JOIN pg_class child ON pg_inherits.inhrelid = child.oid
+            JOIN pg_namespace nmsp_parent ON nmsp_parent.oid = parent.relnamespace
+            JOIN pg_namespace nmsp_child ON nmsp_child.oid = child.relnamespace
+        WHERE parent.oid = '_prom_catalog.series'::regclass
+    ) AS num_series_approx,
+    (
+        SELECT count(*) FROM _prom_catalog.metric
+    ) AS num_metric,
+    (
+        SELECT count(*) FROM _prom_catalog.label_key
+    ) AS num_label_keys,
+    (
+        SELECT count(*) FROM _prom_catalog.label
+    ) AS num_label_pairs;
+
+CREATE OR REPLACE VIEW prom_info.metric_cardinality AS
+    SELECT metric_name, row_count(format('prom_series.%I', table_name)::regclass)  AS cardinality
+    FROM _prom_catalog.metric ORDER BY cardinality DESC;
+
+CREATE OR REPLACE VIEW prom_info.label_key_cardinality AS
```

Comment:
I  think the right query here is actually 
`SELECT k.key AS label_key, coalesce(cnt,0)
  FROM _prom_catalog.label_key k
  LEFT JOIN (SELECT l.key key, count(*) cnt  FROM _prom_catalog.label l GROUP BY key) as l ON (k.key = l.key)`

The reason the above query is better is because it's a single scan of the label table